src/modules/land_detector/VtolLandDetector.cpp

```diff
@@ -78,9 +78,7 @@ bool VtolLandDetector::_get_ground_contact_state()
 
 bool VtolLandDetector::_get_maybe_landed_state()
 {
-
-	// TODO
-	return false;
+	return MulticopterLandDetector::_get_maybe_landed_state();
```

Comment:
VtolLandDetector is a MulticopterLandDetector. If you want this behavior then simply delete VtolLandDetector::_get_maybe_landed_state() and VtolLandDetector::_get_ground_contact_state() while you're at it.